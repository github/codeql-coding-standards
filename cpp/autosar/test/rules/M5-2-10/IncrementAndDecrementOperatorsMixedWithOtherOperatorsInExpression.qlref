rules/M5-2-10/IncrementAndDecrementOperatorsMixedWithOtherOperatorsInExpression.ql