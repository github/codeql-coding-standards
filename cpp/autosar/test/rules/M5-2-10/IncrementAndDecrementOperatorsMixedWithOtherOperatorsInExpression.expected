| test.cpp:4:8:4:18 | ... + ... | Mixed use of the $@ operator with the $@ operator. | test.cpp:4:8:4:11 | ... ++ | increment (++) | test.cpp:4:8:4:18 | ... + ... | + |
| test.cpp:4:8:4:18 | ... + ... | Mixed use of the $@ operator with the $@ operator. | test.cpp:4:15:4:18 | ... -- | decrement (++) | test.cpp:4:8:4:18 | ... + ... | + |
| test.cpp:5:8:5:18 | ... * ... | Mixed use of the $@ operator with the $@ operator. | test.cpp:5:8:5:11 | ++ ... | increment (++) | test.cpp:5:8:5:18 | ... * ... | * |
| test.cpp:5:8:5:18 | ... * ... | Mixed use of the $@ operator with the $@ operator. | test.cpp:5:15:5:18 | -- ... | decrement (++) | test.cpp:5:8:5:18 | ... * ... | * |
